.circleci/common.sh

```diff
@@ -96,13 +96,14 @@ function run_torch_xla_tests() {
   pushd $XLA_DIR
     echo "Running Python Tests"
     ./test/run_tests.sh
+    # only run test_autocast for cpu and gpu on circleCI.
+    python test/test_autocast.py
```

Comment:
I'm trying to understand the CI scripts - is the idea that because this will now run for both cpu and gpu, this is connected to (and will now run on) the pytorch/pytorch CI?

Reply:
Yes, I used to only run this test for our gpu CI because that is the only intended use case.  However pytorch upstream only run our CPU CI and I want to have coverage for this test on the upstream hence making this change.

We have an internal TPU CI that will use `/test/run_tests.sh`. I am not planning to run this test for TPU so I only add this test to the circleci script.

Reply:
Got it, thanks - I think I follow now. Our CI calls into the scripts here through https://github.com/pytorch/pytorch/blob/15eec8e1d1ea5b3354bc305f1afe0c01a64ea748/.jenkins/pytorch/test.sh#L400. But it doesn't run on a CI machine gpu capability so we only run CPU xla tests.